DigitalLearningSolutions.Web/Styles/trackingSystem/centreAdministrators.scss

```diff
@@ -0,0 +1,31 @@
+﻿@import "~nhsuk-frontend/packages/core/all";
+
+button.nhsuk-pagination__link {
+  @include mq($until: 1024px) {

```

Comment:
Not in the nhs styles, 1024 doesn't appear anywhere. The breakpoints are
mobile:  320px,
tablet:  641px,
desktop: 769px,
large-desktop: 990px

This is only really needed because the 3/4 column layout effectively becomes a tablet at a larger window size. There is definitely an argument for making these other random values we have in the css defined properly as part of a large css refactor (I've done a little for the search/filter/sort/pagination on one of my other tickets)

Reply:
We should try and use the NHS BPs, otherwise we'll end up with loads of them across the project (four is already quite a few). If in this case it's an issue due to the 3/4 column and we've tried the NHS values and they don't work, let's add a comment at the top of the file, just to make the user of the styles aware/avoid someone trying to refactor this if we've already tried. :)